EIPS/eip-6963.md

```diff
@@ -38,171 +38,189 @@ The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL
 
 ```typescript
 /**
- * Represents the assets needed to display a wallet
+ * The information needed for a Dapp to identify and display a Wallet Provider
  */
-interface EIP6963ProviderInfo {
+interface Eip6963ProviderInfo {
   uuid: string;
   name: string;
   icon: string;
   rdns: string;
 }
 ```
 
-#### Images/Icons
+The `Eip6963ProviderInfo` object MAY also be extended with additional properties. If a Dapp does not recognize the additional properties, it SHOULD ignore them.
+
+#### Icon images
 
-A URI-encoded image was chosen to enable flexibility for multiple protocols for fetching and rendering icons, for example:
+The `icon` string MUST be a data URI as defined in [RFC-2397]. The image SHOULD be a square with a minimum resolution of 96x96px. The image format is RECOMMENDED to be either lossless or vector-based such as PNG, WebP or SVG to make it easily renderable by the Dapp.
+
+URI-encoded images are flexible and allow for multiple protocols for fetching and rendering icons, for example:
 
 ```sh
-# svg (data uri)
+# SVG (data URI)
 data:image/svg+xml,<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="32px" height="32px" viewBox="0 0 32 32"><circle fill="red" cx="16" cy="16" r="12"/></svg>
-# png (data uri)
+# PNG (data URI)
 data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4//8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU5ErkJggg==
 ```
 
-The `icon` string MUST be a data URI as defined in [RFC-2397]. The image SHOULD be a square with 96x96px minimum resolution. The image format is RECOMMENDED to be either lossless or vector based such as PNG, WebP or SVG to make the image easy to render on the DApp. Since SVG images can execute Javascript, applications and libraries MUST render SVG images using the `<img>` tag to ensure no untrusted Javascript execution can occur.
+Since SVG images can execute JavaScript, Dapps and Wallet Provider Discovery Libraries MUST render SVG images using the `<img>` tag to ensure no untrusted JavaScript execution can occur.
 
-#### RDNS
+#### RDNS identifiers
 
-The **`rdns`** (Reverse-DNS) property serves to provide an identifier which DApps can rely on to be stable between sessions. The Reverse Domain Name Notation is chosen to prevent namespace collisions.
-The Reverse-DNS convention implies that the value should start with a reversed DNS domain name controlled by the Provider. The domain name should be followed by a subdomain or a product name. Example: `com.example.MyBrowserWallet`.
+The `rdns` (reverse-DNS) property serves as an identifier which Dapps can rely on to be stable between page sessions. The Reverse Domain Name Notation is chosen to prevent namespace collisions. The domain should be controlled by the Wallet Provider and be followed by a subdomain or a product name. Example: `com.example.MyBrowserWallet`.
```

Comment:
```suggestion
The `rdns` (reverse-DNS) property serves as an identifier which Dapps can rely on to be stable between page sessions. The Reverse Domain Name Notation is chosen to prevent namespace collisions. The domain SHOULD be controlled by the Wallet Provider and be followed by a subdomain or a product name. Example: `com.example.MyBrowserWallet`.
```